File: Examples/VisualBasic/AsposePDF/DocumentConversion/EPUBToPDF.vb
Imports System.IO
Imports System
Imports Microsoft.VisualBasic
Imports Aspose.Pdf

Namespace AsposePDF.DocumentConversion
    Public Class EPUBToPDF
        Public Shared Sub Run()
            Try
                ' ExStart:EPUBToPDF
                ' The path to the documents directory.
                Dim dataDir As String = RunExamples.GetDataDir_AsposePdf_DocumentConversion()

                ' Instantiate LoadOption object using EPUB load option
                Dim epubload As New EpubLoadOptions()

                ' Create Document object
                Dim pdf As New Document(dataDir & Convert.ToString("EPUBToPDF.epub"), epubload)

                ' Save the resultant PDF document
                pdf.Save(dataDir & Convert.ToString("EPUBToPDF_out_.pdf"))
                ' ExEnd:EPUBToPDF
            Catch ex As Exception
                Console.WriteLine(ex.Message)
            End Try
        End Sub
    End Class
End Namespace

File: Examples/CSharp/AsposePDF/DocumentConversion/EPUBToPDF.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;

namespace CSharp.AsposePdf.DocumentConversion
{
    public class EPUBToPDF
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdf_DocumentConversion();

            // Instantiate LoadOption object using EPUB load option
            EpubLoadOptions epubload = new EpubLoadOptions();

            // Create Document object
            Aspose.Pdf.Document pdf = new Aspose.Pdf.Document(dataDir + "Sample.epub", epubload);
            
            // Save the resultant PDF document
            pdf.Save(dataDir + "EPUBToPDF_out.pdf");
        }
    }
}